pkgs/development/python-modules/anchor/default.nix

```
@@ -5,7 +5,7 @@
   setuptools,
 }:
 buildPythonPackage {
-  pname = "anchor-kr";
+  pname = "anchor";
```

```suggestion
  pname = "anchor-kr";
```
https://pypi.org/project/anchor-kr/